packages/layouts-react/src/components/PageLayout/Left.tsx

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
```

Comment:
You can combine all ActionBar subcomponents into ActionBar file. It would take less space (eg. what we are doing with Flex, FileUploadCard)

Reply:
I have combined all ActionBar subcomponents and deleted the old ones.